etc/HIReplayOfflineConfiguration.py

```diff
@@ -1433,7 +1469,7 @@
                timePerEvent=1,
                disk_node="T2_US_Vanderbilt",
                dqm_sequences=["@commonSiStripZeroBias"],
-               scenario=hiTestppScenario)
+               scenario=hiLightEventsScenario)
```

Comment:
@LinaresToine I don't understand this choice (and also several others).
In 2023 this PD was processed with `ppEra_Run3_pp_on_PbPb_2023`. I think this should be `hiHeavyEventsScenario` (= `"ppEra_Run3_pp_on_PbPb_2024"`) now.